anyway/widgets/suburban_widgets/accident_count_by_car_type_widget.py

```diff
@@ -28,95 +33,117 @@ def generate_items(self) -> None:
+                table_obj=VehicleMarkerView,
+                filters=request_params.location_info,
+                group_by="vehicle_type",
+                count="provider_and_id",
                 start_time=request_params.start_time,
                 end_time=request_params.end_time,
             )
 
         start_time = request_params.start_time
         end_time = request_params.end_time
         data_by_segment = AccidentCountByCarTypeWidget.percentage_accidents_by_car_type(
-            involved_by_vehicle_type_data
+            vehicle_grouped_by_type_count_unique
         )
         national_data = AccidentCountByCarTypeWidget.percentage_accidents_by_car_type_national_data_cache(
             start_time, end_time
         )
 
-        for k, v in national_data.items():  # pylint: disable=W0612
+        for car_type in data_by_segment:
             out.append(
                 {
-                    "car_type": k,
-                    "percentage_segment": data_by_segment[k],
-                    "percentage_country": national_data[k],
+                    "label_key": car_type,
+                    "series": [
+                        {"label_key": "percentage_segment", "value": data_by_segment[car_type]},
+                        {"label_key": "percentage_country", "value": national_data[car_type]},
+                    ],
                 }
             )
         return out
 
     @staticmethod
-    def percentage_accidents_by_car_type(involved_by_vehicle_type_data):
-        driver_types = defaultdict(float)
+    def percentage_accidents_by_car_type(vehicle_grouped_by_type_count_unique) -> Dict[str, int]:
+        vehicle_type_dict = defaultdict(int)
         total_count = 0
-        for item in involved_by_vehicle_type_data:
-            vehicle_type, count = item["involve_vehicle_type"], int(item["count"])
+        for item in vehicle_grouped_by_type_count_unique:
+            vehicle_type, count = item["vehicle_type"], int(item["count"])
             total_count += count
             if vehicle_type in VehicleCategory.CAR.get_codes():
-                driver_types[VehicleCategory.CAR.value] += count
+                vehicle_type_dict[VehicleCategory.CAR.value] += count
             elif vehicle_type in VehicleCategory.LARGE.get_codes():
-                driver_types[VehicleCategory.LARGE.value] += count
+                vehicle_type_dict[VehicleCategory.LARGE.value] += count
             elif vehicle_type in VehicleCategory.MOTORCYCLE.get_codes():
-                driver_types[VehicleCategory.MOTORCYCLE.value] += count
+                vehicle_type_dict[VehicleCategory.MOTORCYCLE.value] += count
             elif vehicle_type in VehicleCategory.BICYCLE_AND_SMALL_MOTOR.get_codes():
-                driver_types[VehicleCategory.BICYCLE_AND_SMALL_MOTOR.value] += count
+                vehicle_type_dict[VehicleCategory.BICYCLE_AND_SMALL_MOTOR.value] += count

```

Comment:
I think electric scooter is not included in any of the categories here. Is it on purpose?

Reply:
It wasn't part of the requirements, i can add it, @atalyaalon what do you think?

Reply:
@BarVolunteering @ziv17 it's included in the BICYCLE_AND_SMALL_MOTOR category. (see vehicle_type.py)

Reply:
let's discuss in the meeting